CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_tiktok_ads v0.5.0
```

Comment:
Should we also include a callout regarding the renamed identifiers from the source? Would we be able to make a breaking change callout in this changelog to refer to the source release notes for details around breaking changes. Or is there a better approach you can think of to inform our users of the breaking changes in the upstream dependent package.